docs/middlewares/dedupleMiddleware.docs.md

```diff
@@ -0,0 +1,36 @@
+# [Deduple middleware](https://github.com/botify-labs/botify-sdk-js-middlewares/blob/master/src/middlewares/dedupleMiddleware.js)
+
+When possible, this middleware will merge operation if they have the same params, this permit to reduce the number of call made to the API.
+
+If a request is merged it don't continue the middleware chain, so be carefull with the order.
```

Comment:
Middlewares chain of duplicate operations are interrupted. So you might consider middlewares order carefully.